Example/Tests/GINIOpenWithTutorialViewControllerTests.swift

```diff
@@ -0,0 +1,113 @@
+        
+        XCTAssertEqual(cell.stepIndicator.text, String(describing: indexPath.row + 1), "step indicator for second step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepTitle.text, item.title, "step title for second step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepSubTitle.text, item.subtitle, "step subtitle for second step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepImage.image, item.image, "step image for second step should be the same as the one declared on initialiation")
+    }
+    
+    func testThirdStepProperties() {
+        let indexPath = IndexPath(row: 2, section: 0)
+        let item = openWithTutorialViewController.items[indexPath.row]
+        
+        let cell = openWithTutorialViewController.collectionView(openWithTutorialViewController.collectionView!, cellForItemAt: indexPath) as! OpenWithTutorialCollectionCell
+        
+        XCTAssertEqual(cell.stepIndicator.text, String(describing: indexPath.row + 1), "step indicator for third step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepTitle.text, item.title, "step title for third step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepSubTitle.text, item.subtitle, "step for third step subtitle should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepImage.image, item.image, "step image for third step should be the same as the one declared on initialiation")
+    }
+    
+    func testSixthMockedStepProperties() {
+        openWithTutorialViewController.items = mockedItems
+        let indexPath = IndexPath(row: 5, section: 0)
+        let item = openWithTutorialViewController.items[indexPath.row]
+        
+        let cell = openWithTutorialViewController.collectionView(openWithTutorialViewController.collectionView!, cellForItemAt: indexPath) as! OpenWithTutorialCollectionCell
+        
+        XCTAssertEqual(cell.stepIndicator.text, String(describing: indexPath.row + 1), "step indicator for sixth mocked step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepTitle.text, item.title, "step title for sixth mocked step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepSubTitle.text, item.subtitle, "step subtitle for sixth mocked step should be the same as the one declared on initialiation")
+        XCTAssertEqual(cell.stepImage.image, item.image, "step image for sixth mocked step should be the same as the one declared on initialiation")
+    }
+
+    func testHeaderDissapearOnLandscape() {
+        let collectionView = openWithTutorialViewController.collectionView!
+        openWithTutorialViewController.view.frame.size = CGSize(width: 1, height: 0) // Simulate landscape
```

Comment:
Manipulating the frame size is not something I'd recommend for orientation changes. It's just a side effect.
How about using the view controller orientation change stuff? Is it possible?

Reply:
I don't like this approach but first I tried to force device/simulator rotation (didn't work), and what I think you suggest (`viewWillTransitionToSize...`) doesn't rotate the interface.
Do you have any better idea? Or do you think we should remove this test since It could be more a UITest?